src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable linebreak-style */
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Based on the requirements of this project, the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK`. Kindly do the fix :wrench: